docs/development/contributing.md

```diff
@@ -40,14 +40,15 @@ mail to the [Technical Board of the ILIAS society](mailto:tb@lists.ilias.de).
 ### Pull Request to the Repositories
 
 Pull requests (PRs) will be assigned by the [Technical Board
```

Comment:
Do we always need the TB to assign PRs?

Reply:
No, I don't think this is fully right. If added a condition.